README.md

```diff
@@ -165,7 +165,7 @@ We use [GoReleaser](https://goreleaser.com/) to build and upload binaries to our
 Summary:
 - Test the build locally:
   ```
-  make release-local
+  go run github.com/go-task/task/v3/cmd/task release-local
```

Comment:
You'll need to update `make test` above as well. I used that the other day to make the `GeoIP2-Country-Test.mmdb` file as well, which is needed in tests. Maybe add that into the README as well?

Reply:
Fixed the README. The task already sets up the submodule for you.

Reply:
Ah yeah I see now, I'm new to Task.